src/containers/Layout/Layout.tsx

```diff
@@ -45,7 +54,7 @@ const Layout = () => {
   } = useProfiles();
 
   if (profilesEnabled && !profiles?.length) {
-    unpersistProfile();
+    profileController?.unpersistProfile();
```

Comment:
Am I missing something? Shouldn't this be in an effect or something?

Reply:
Good catch. I see that it is a local storage interaction, but we still should not trigger it that way.